main.js

```diff
@@ -0,0 +1,127 @@
+// 1. Створити функцію-конструктор Людини. Властивості, які будуть описувати екземпляр Людини - ім'я, вік, стать, національність, країна, список країн для подорожей. Створити універсальні функції - знайомство, прокидатися, засинати й список країн, які Людина хоче відвідати. Роботу цих функцій можна виводити через console.log. Але в кожній з функцій повинно використовуватись якомога більше характеристик конкретної Людини на якій ця функція викликається (підказка: тут треба використати call/apply)
+
+function CreatePerson (name, age, sex, nationality, country, countriesForVisiting) { 
+    this.name = name; 
+    this.age = age; 
+    this.sex = sex; 
+    this.nationality = nationality; 
+    this.country = country; 
+    this.countriesForVisiting = countriesForVisiting;   
+  }
+
+const acquaintance = function () { 
+    console.log(`Hi. My name is ${this.name}, I'm from ${this.country}. Here is a list of countries that I'm going to visit: ${this.countriesForVisiting}!`) 
+  }; 
+const wakeUp = function () {
+    console.log(`Hey, ${this.name} wake up! We're going back home to ${this.country}! I missed my family so much.. `)
+}
+const fallAsleep = function () {
+    console.log(`Why doesn't ${this.name} go to sleep? She has to wake up early tomorrow because she has a train to ${this.country}!`)
+}
+const listCountiesToGo = function () {
+    console.log(`${this.name} has been dreaming to visit ${this.countriesForVisiting} till she became ${this.age}.`)
+}
+
+const me = new CreatePerson('Anna', 30, 'female', 'ukrainian', 'Ukraine', ['England, France, Switzerland']); 
+
+console.log(me); 
+
+acquaintance.apply(me, [me.name, me.country, me.countriesForVisiting]);
+wakeUp.apply(me, [me.name, me.country]);
+fallAsleep.call(me, me.name, me.country);
+listCountiesToGo.bind(me)(me.name, me.countriesForVisiting, me.age);
```

Comment:
It should be enough to call/apply something just for obj without args
`acquaintance.apply(me)`